### ISSUE TITLE
Add to e2e test adding all default plugins to velero deployment doesn't cause image pull errors

### ISSUE THREAD

**Issue by kaovilai** [openshift/oadp-operator#631]

**Comment by mateusoliveira43** [comment]
What are all default images? these? https://github.com/openshift/oadp-operator/blob/master/config/manager/manager.yaml#L39-L56

how should be a test to check they don't conflict with each other?

**Comment by kaovilai** [comment]
> What are all default images? these? https://github.com/openshift/oadp-operator/blob/master/config/manager/manager.yaml#L39-L56

yes. Adding them all at once should cause add all the plugins to container and event/pod should tell us if there are image pull errors. Though this may only prove that the upstream quay images are pullable.

> how should be a test to check they don't conflict with each other?

I was thinking that [each plugin initcontainer essentially copy dirs into /plugin folder](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/67653e58ce9f461ba79736c2919d9a017747b755/Dockerfile#L32C1-L34C51)
```go
FROM scratch
COPY --from=build /go/bin/velero-plugin-for-aws /plugins/
COPY --from=build /go/bin/cp-plugin /bin/cp-plugin
```
It would be wise to check that no more than one plugin uses path `velero-plugin-for-aws` for example. In other words,  if there are 5 plugins, you would expect there to be 5 dirs under `/plugins/` in velero pod (in addition to any velero core plugins if any).
